showcase-docs/dynamic-plugins.md

```diff
@@ -218,7 +226,7 @@ Examples of such a wrapper plugins can be found in the [Janus showcase repositor
           disabled: true
   ```
 
-- A plugin of an included file, disabled by default, can be enabled with the following snippet:
+- A plugin disabled in an included file can be enabled with the following snippet:
```

Comment:
@davidfestal can I ask if this is what you meant in the previous documentation?

Reply:
That was it, and thanks for making it much clearer :-)